osm-fudge/n_gram.py

```diff
@@ -0,0 +1,15 @@
+'''N-gram function
+
+Find n-grams of a given string
+
+More: https://en.wikipedia.org/wiki/N-gram
+'''
+
+import sys
+
+def find_n_grams(n, str):
+    return [str[i:i+n] for i in range(len(str) - (n-1))]
```

Comment:
I think we should put pre- and post-conditions here:
- Which values for n make sense? => assert
- How many n-grams to we expect for a string of length l? => assert